features/talk.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Talk Selection
+    In order to test CRUD on Talk
+    As an admin
+    I need to be able to list all task and fill forms
+
+  Scenario: I don't add an talk and go back to home
+    Given I am logged in as an admin
+    When I am on the talk creation page
+    And I fill the talk "title" field with "My best Talk dependency injection"
+    And I fill the talk "description" field with "In this talk i can speak the dependancy injection"
+    And I fill the talk "speaker" field with "82e7325c-36b3-4c33-a0e8-743c6013e008"
```

Comment:
`82e7325c-36b3-4c33-a0e8-743c6013e008` is too technical, can we have a business name rather than that?